src/AppBundle/Entity/Adherent.php

```diff
@@ -657,21 +657,15 @@ public function isSupervisorOf(Committee $committee): bool
 
     public function setLegislativeCandidate(?string $area): void
     {
-        if (!$area) {
-            $this->legislative = null;
-
-            return;
+        if (!$this->legislative) {
```

Comment:
🤔 If I create an adherent and execute `setLegislativeCandidate(null)` it will not go in the if and will play the `setArea`. No ?